### PR TITLE
Fix groups allowed to validate indicators

### PR DESCRIPTION
**Pull request by ghost**
Program director may make the first validation. Program dpe may make the second validation. The manager may do both.

### CODE REVIEW

File: program_indicator/security/ir_rule.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <field model="ir.model" name="model_id" ref="model_program_result_indicator_value"/>
+        <field name="domain_force">[('state', '=', 'draft')]</field>
+        <field name="groups" eval="[(4, ref('program.group_program_director'))]"/>
+        <field name="perm_read" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+    </record>
+
+    <record id="rule_indicator_value_dpe" model="ir.rule">
+        <field name="name">Program Indicator Value DPE</field>
```
Comment (yvaucher) [inline program_indicator/security/ir_rule.xml:16]: What is DPE? Wouldn't it be better to have the full name?

Comment (ghost) [inline program_indicator/security/ir_rule.xml:16]: @yvaucher, I agree 100% with you. This module was developed with security rules specific to the needs of one client. It does not fit in OCA in its current state. DPE is an expression used by one of Savoir-faire Linux's client. One day, I will propose another module to remove those security rules from OCA modules.

**Comment by gurneyalex** [comment]
@dufresnedavid you get my :+1: if you commit to fixing this to remove the specificity.

**Comment by ghost** [comment]
@gurneyalex @yvaucher I plan to go even further and remove completely the modules program from OCA. These modules are extremely specific to one company.  They were not designed to be usable universally. It is unlikely that someone would install this set of modules and figure out how it works in a reasonable time.

**Pull request merged**